usagov/vote-gov-drupal
# VOTE-1114: Install and Enable log_stdout contrib module with patch

## Jira ticket (required)
[VOTE-1114](https://bixal-projects.atlassian.net/browse/VOTE-1114)

## Description (optional)
Install and enable log_stdout contrib module.  The module needed to have an issue fork created in Drupal.org, in addition to a patch applied to work in Drupal 10.

## Deployment and testing (required, if applicable)

### Testing steps
1. Log into site and bring up Docker appserver container.
2. Save a node and perform other loggable tasks.
3. Verify messages in the Drupal watchdog are reflected in json format in the Docker appserver log.
